docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Host and deploy Blazor
+author: guardrex
+description: Discover how to host and deploy Blazor apps using ASP.NET Core, Content Delivery Networks (CDN), file servers, and GitHub Pages.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/host-and-deploy/index
+---
+# Host and deploy Blazor
+
+By [Luke Latham](https://github.com/guardrex) and [Rainer Stropek](https://www.timecockpit.com)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+## Publishing
+
+Blazor apps are published for deployment in Release configuration with the [dotnet publish](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish) command:
+
+```console
+dotnet publish -c Release
+```
+
+The .NET linker strips unused [Intermediate Language (IL)](https://docs.microsoft.com/dotnet/standard/managed-code) when the app is built in Release configuration.
+
+The deployment is created in the */bin/Release/netstandard2.0/publish* folder. The assets in the *publish* folder are moved to the web server.
```

Comment:
> The assets in the *publish* folder are moved to the web server.

That's misleading as it implies it happens automatically. Just running `dotnet publish` doesn't do this. I'm guessing you intend to say something like "The contents of the `publish` directory are what you need to deploy to your production web server".

Reply:
> That's misleading as it implies it happens automatically.

Yep ... let's make a new section for this remark called *Deployment*. We can expand the language from there on the next commit if needed.